plugins/nodes/src/js/pages/nodes-overview/HostsPageContent.js

```diff
@@ -154,6 +156,33 @@ class HostsPageContent extends React.Component {
       </div>
     );
   }
+  getGridViewResourceSwitch() {
+    if (!this.props.location.pathname.includes("grid")) {
+      // we have to return an empty div here (instead of null),
+      // because FilterBar has fixed rightAlignLastNChildren={2}
+      return <div />;
+    }
+
+    const resourceColors = ResourcesUtil.getResourceColors();
+    const resourceLabels = ResourcesUtil.getResourceLabels();
+    const buttons = ResourcesUtil.getDefaultResources().map(resource => {
+      const classSet = classNames("button button-outline", {
```

Comment:
🎨Funny that I ask you that right now, but can we maybe inline the classSet and use the classnames package for the entire string concatenation there? It seems odd to me to also use template strings here.

Reply:
I think the way this package is used in out codebase it is "correct" this way. having all the default classes as string and only adding optional ones from the object… 🤔

Reply:
Yeah, the package usage is correct, but below you do `className={`${classSet} path-color-${resourceColors[resource]}`}`. You could put the `path-color-${resourceColors[resource]}` also into classNames, that woul be easier

Reply:
true. 🤷‍♂️  (legacy code, just copy pasted…)